bsd/models.py

```diff
@@ -45,7 +45,8 @@ def assert_valid_account(api_result):
 
 def find_constituents_by_state_cd(state_cd):
```

Comment:
i suggest to add optional cons group param to method signature. this feels too much hardcoded logic. when we get list for calling tool, i dont think we care about sendable cons group right?

Reply:
Agreed.